### PR TITLE
[DNM] kvrpc: move context to Request

### PR DESCRIPTION
**Pull request by disksing**
@ngaut @siddontang @morefreeze @BusyJay PTAL

### CODE REVIEW

**Comment by BusyJay** [comment]
LGTM

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -150,23 +143,24 @@ message CmdCommitThenGetResponse {
...
-    optional CmdRollbackThenGetRequest cmd_rb_get_req   = 7;
-    optional CmdCommitThenGetRequest cmd_commit_get_req = 8;
+    optional MessageType               type               = 1;
+    optional Context                   context            = 2;
+    optional CmdGetRequest             cmd_get_req        = 3;
+    optional CmdScanRequest            cmd_scan_req       = 4;
+    optional CmdPrewriteRequest        cmd_prewrite_req   = 5;
+    optional CmdCommitRequest          cmd_commit_req     = 6;
+    optional CmdCleanupRequest         cmd_cleanup_req    = 7;
+    optional CmdRollbackThenGetRequest cmd_rb_get_req     = 8;
```
Comment (qiuyesuifeng) [inline proto/kvrpcpb.proto]: How about directly using `cmd_rollback_get_req`?
Seems much clearer.

**Comment by disksing** [comment]
PTAL @qiuyesuifeng

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request closed without merging**